docs/learn/learn-staking.md

```diff
@@ -321,9 +321,9 @@ validator. The penalty would be Min(3 \* 1 / 100)^2, 1) = 0.0009, or a 0.09% sla
 validator pool (i.e., all stake held by the validator and its nominators).
 
 Now assume that there is a group running several validators, and all of them have an issue in the
-same slot. The penalty would be Min((3 _ 5 / 100)^2, 1) = 0.0225, or a 2.25% slash. If 20 validators
+same slot. The penalty would be Min((3 \_ 5 / 100)^2, 1) = 0.0225, or a 2.25% slash. If 20 validators
```

Comment:
I don't understand the usage of underscore symbol here. Isn't it supposed to be ((3 * 5)/100)^2 which is equal to 0.0225 and min(0.0225, 1) = 0.0225

Reply:
Using \* instead of \_ might fix the issue and stay true to the equation

Reply:
Addressed.